ckanext/canada/schemas/dataset.yaml

```diff
@@ -78,21 +78,25 @@ dataset_fields:
     en: Parent identifier
     fr: Identifiant parent
   form_snippet: null
+  validators: string_safe
```

Comment:
I like to use `scheming_required` https://github.com/ckan/ckanext-scheming/blob/3aa6ed7f134d14b882c5cea474d6519eed36bac9/ckanext/scheming/validation.py#L165 as the first validator most of the time because if `required:true` is not set it acts as `ignore_missing` which saves the `if value is None or value is missing: return` dance at the beginning of all the new validators you added.

`scheming_required` also actually enforces requiredness when `required:true` is added to the schema

Reply:
@wardi okay I have removed those code checks for empty values, and added some scheming_required/ignore_missing on a bunch of the fields.

I also added in the new validator for the licence_choices.